### ISSUE TITLE
HUD Not Toggleable

### ISSUE THREAD

**Issue by kileraptor1** [AxTo/GlassChat3#14]
It just stacks it, increasing the opacity until you can't see the chat anymore.

**Comment by Zyiks** [comment]
Alex pls

**Comment by lesander** [comment]
Hi @kileraptor1. The `master` branch is outdated and could cause trouble on newer versions of the mods. Please try using v3.1.0: https://github.com/AxTo/GlassChat3/tree/3.1.0 and see if your issue still occurs.

**Comment by Zyiks** [comment]
The issue can't occur on the branch if the command is not available on that branch.

Alex pls

**Comment by lesander** [comment]
Okay, I'll bite. Please elaborate @Zyiks, I'm not quite sure what you mean.

**Comment by Zyiks** [comment]
On the branch 3.1.0 there is no "gc hud" command, aka upgrading will just remove the broken command(currently it would have constantly made new opaque boxes in the same spot, stacking them would increase opacity) so the issue can not occur.

This commit removed the hud command from 3.1.0
https://github.com/AxTo/GlassChat3/commit/6aa85326ad9a3d6cf5617d5a50dd9adb790459f9

**Comment by lesander** [comment]
Right. Well, the `master` branch doesn't work with the current versions CC and OpenMods (as this issue demonstrates), but the `3.1.0` branch does. My advice is to upgrade to the `3.1.0` branch. Yes, you'll lose the `gc hud` command, but everything else should work.

Perhaps I'll add the `gc hud` command in the future to `3.1.0`.
